[2020-02-25 19:26:26,326] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:26:26,683] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:26:28,726] - [rest_client] [123145331109888] - INFO - Node version in cluster 7.0.0-1288-enterprise
[2020-02-25 19:26:28,803] - [rest_client] [123145331109888] - INFO - Node versions in cluster ['7.0.0-1288-enterprise']
[2020-02-25 19:26:28,804] - [basetestcase] [123145331109888] - INFO - ==============  basetestcase setup was started for test #1 test_role_permission_validate_multiple==============
[2020-02-25 19:26:29,658] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.65:8091
[2020-02-25 19:26:29,706] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.65:8091 is running
[2020-02-25 19:26:29,764] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.67:8091
[2020-02-25 19:26:29,803] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.67:8091 is running
[2020-02-25 19:26:29,843] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.66:8091
[2020-02-25 19:26:29,891] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.66:8091 is running
[2020-02-25 19:26:29,929] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.68:8091
[2020-02-25 19:26:29,987] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.68:8091 is running
[2020-02-25 19:26:30,103] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.104.28:8091
[2020-02-25 19:26:30,151] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.104.28:8091 is running
[2020-02-25 19:26:30,248] - [basetestcase] [123145331109888] - INFO - ------- Cluster statistics -------
[2020-02-25 19:26:30,249] - [basetestcase] [123145331109888] - INFO - 172.23.121.65:8091 => {'services': ['kv'], 'cpu_utilization': 3.015075376884422, 'mem_free': 3285782528, 'mem_total': 4366938112, 'swap_mem_used': 43032576, 'swap_mem_total': 3758092288}
[2020-02-25 19:26:30,249] - [basetestcase] [123145331109888] - INFO - --- End of cluster statistics ---
[2020-02-25 19:26:30,250] - [basetestcase] [123145331109888] - INFO - ==============  basetestcase cleanup was started for test #1 test_role_permission_validate_multiple ==============
[2020-02-25 19:26:31,024] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.65:8091
[2020-02-25 19:26:31,081] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.65:8091 is running
[2020-02-25 19:26:31,128] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.67:8091
[2020-02-25 19:26:31,166] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.67:8091 is running
[2020-02-25 19:26:31,204] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.66:8091
[2020-02-25 19:26:31,252] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.66:8091 is running
[2020-02-25 19:26:31,290] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.121.68:8091
[2020-02-25 19:26:31,347] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.121.68:8091 is running
[2020-02-25 19:26:31,396] - [cluster_helper] [123145331109888] - INFO - waiting for ns_server @ 172.23.104.28:8091
[2020-02-25 19:26:31,446] - [cluster_helper] [123145331109888] - INFO - ns_server @ 172.23.104.28:8091 is running
[2020-02-25 19:26:31,447] - [ntonencryptionBase] [123145331109888] - INFO - Disable up node to node encryption - status = disable and clusterEncryptionLevel = control
[2020-02-25 19:26:31,495] - [rest_client] [123145331109888] - INFO - settings/autoFailover params : timeout=120&enabled=false&failoverOnDataDiskIssues%5Benabled%5D=false&maxCount=1&failoverServerGroup=false
[2020-02-25 19:26:31,591] - [rest_client] [123145331109888] - INFO - settings/autoFailover params : timeout=120&enabled=false&failoverOnDataDiskIssues%5Benabled%5D=false&maxCount=1&failoverServerGroup=false
[2020-02-25 19:26:31,715] - [rest_client] [123145331109888] - INFO - settings/autoFailover params : timeout=120&enabled=false&failoverOnDataDiskIssues%5Benabled%5D=false&maxCount=1&failoverServerGroup=false
[2020-02-25 19:26:31,812] - [rest_client] [123145331109888] - INFO - settings/autoFailover params : timeout=120&enabled=false&failoverOnDataDiskIssues%5Benabled%5D=false&maxCount=1&failoverServerGroup=false
[2020-02-25 19:26:31,908] - [rest_client] [123145331109888] - INFO - settings/autoFailover params : timeout=120&enabled=false&failoverOnDataDiskIssues%5Benabled%5D=false&maxCount=1&failoverServerGroup=false
[2020-02-25 19:26:31,984] - [ntonencryptionBase] [123145331109888] - INFO - Changing encryption Level - clusterEncryptionLevel = control
[2020-02-25 19:26:31,987] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:26:32,333] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:26:34,898] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli setting-security -c http://localhost -u Administrator -p password --set --cluster-encryption-level control
[2020-02-25 19:26:35,173] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:26:35,470] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.67
[2020-02-25 19:26:38,067] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli setting-security -c http://localhost -u Administrator -p password --set --cluster-encryption-level control
[2020-02-25 19:26:38,359] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:26:38,707] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.66
[2020-02-25 19:26:40,206] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli setting-security -c http://localhost -u Administrator -p password --set --cluster-encryption-level control
[2020-02-25 19:26:40,429] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:26:40,712] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.68
[2020-02-25 19:26:42,974] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli setting-security -c http://localhost -u Administrator -p password --set --cluster-encryption-level control
[2020-02-25 19:26:43,259] - [remote_util] [123145331109888] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:26:43,588] - [remote_util] [123145331109888] - INFO - Connected to 172.23.104.28
[2020-02-25 19:26:46,148] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli setting-security -c http://localhost -u Administrator -p password --set --cluster-encryption-level control
[2020-02-25 19:26:46,427] - [ntonencryptionBase] [123145331109888] - INFO - Output of setting-security command is ["ERROR: clusterEncryptionLevel - Can't set cluster encryption level when cluster encryption is disabled."]
[2020-02-25 19:26:46,428] - [ntonencryptionBase] [123145331109888] - INFO - Error of setting-security command is []
[2020-02-25 19:26:46,428] - [ntonencryptionBase] [123145331109888] - INFO - Changing node-to-node-encryption to disable
[2020-02-25 19:26:46,430] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:26:46,764] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:26:49,281] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli node-to-node-encryption -c http://localhost -u Administrator -p password --disable
[2020-02-25 19:26:49,582] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:26:49,916] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.67
[2020-02-25 19:26:52,478] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli node-to-node-encryption -c http://localhost -u Administrator -p password --disable
[2020-02-25 19:26:52,768] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:26:53,089] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.66
[2020-02-25 19:26:55,682] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli node-to-node-encryption -c http://localhost -u Administrator -p password --disable
[2020-02-25 19:26:55,977] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:26:56,303] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.68
[2020-02-25 19:26:58,878] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli node-to-node-encryption -c http://localhost -u Administrator -p password --disable
[2020-02-25 19:26:59,168] - [remote_util] [123145331109888] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:26:59,476] - [remote_util] [123145331109888] - INFO - Connected to 172.23.104.28
[2020-02-25 19:27:01,594] - [remote_util] [123145331109888] - INFO - command to run: /opt/couchbase/bin/couchbase-cli node-to-node-encryption -c http://localhost -u Administrator -p password --disable
[2020-02-25 19:27:01,869] - [ntonencryptionBase] [123145331109888] - INFO - Output of node-to-node-encryption command is ['Turned off encryption for node: http://172.23.104.28:8091', 'SUCCESS: Switched node-to-node encryption off']
[2020-02-25 19:27:01,870] - [ntonencryptionBase] [123145331109888] - INFO - Error of node-to-node-encryption command is []
[2020-02-25 19:27:01,871] - [basetestcase] [123145331109888] - INFO - ==============  basetestcase cleanup was finished for test #1 test_role_permission_validate_multiple ==============
[2020-02-25 19:27:01,985] - [basetestcase] [123145331109888] - INFO - initializing cluster
[2020-02-25 19:27:03,096] - [task] [123145341620224] - INFO - server: ip:172.23.121.65 port:8091 ssh_username:root, nodes/self: {'uptime': '26223', 'memoryTotal': 4366938112, 'memoryFree': 3288109056, 'mcdMemoryReserved': 3331, 'mcdMemoryAllocated': 3331, 'status': 'healthy', 'hostname': '172.23.121.65:8091', 'clusterCompatibility': 458752, 'clusterMembership': 'active', 'version': '7.0.0-1288-enterprise', 'os': 'x86_64-unknown-linux-gnu', 'ports': [], 'availableStorage': [], 'storage': [<membase.api.rest_client.NodeDataStorage object at 0x10c287490>], 'memoryQuota': 2231, 'moxi': 11211, 'memcached': 11210, 'id': 'ns_1@172.23.121.65', 'ip': '172.23.121.65', 'rest_username': '', 'rest_password': '', 'port': '8091', 'services': ['kv'], 'storageTotalRam': 4164, 'curr_items': 0}
[2020-02-25 19:27:03,154] - [rest_client] [123145341620224] - INFO - pools/default params : memoryQuota=2231
[2020-02-25 19:27:03,233] - [rest_client] [123145341620224] - INFO - --> init_node_services(Administrator,password,172.23.121.65,8091,['kv'])
[2020-02-25 19:27:03,234] - [rest_client] [123145341620224] - INFO - /node/controller/setupServices params on 172.23.121.65: 8091:hostname=172.23.121.65&user=Administrator&password=password&services=kv
[2020-02-25 19:27:03,300] - [rest_client] [123145341620224] - ERROR - POST http://172.23.121.65:8091//node/controller/setupServices body: hostname=172.23.121.65&user=Administrator&password=password&services=kv headers: {'Content-Type': 'application/x-www-form-urlencoded', 'Authorization': 'Basic QWRtaW5pc3RyYXRvcjpwYXNzd29yZA==', 'Accept': '*/*'} error: 400 reason: unknown b'["cannot change node services after cluster is provisioned"]' auth: Administrator:password
[2020-02-25 19:27:03,310] - [rest_client] [123145341620224] - INFO - This node is already provisioned with services, we do not consider this as failure for test case
[2020-02-25 19:27:03,311] - [rest_client] [123145341620224] - INFO - --> in init_cluster...Administrator,password,8091
[2020-02-25 19:27:03,312] - [rest_client] [123145341620224] - INFO - settings/web params on 172.23.121.65:8091:port=8091&username=Administrator&password=password
[2020-02-25 19:27:03,406] - [rest_client] [123145341620224] - INFO - --> status:True
[2020-02-25 19:27:03,408] - [remote_util] [123145341620224] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:27:03,771] - [remote_util] [123145341620224] - INFO - Connected to 172.23.121.65
[2020-02-25 19:27:07,862] - [remote_util] [123145341620224] - INFO - running command.raw on 172.23.121.65: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:07,937] - [remote_util] [123145341620224] - INFO - command executed successfully
[2020-02-25 19:27:07,996] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.65:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:08,072] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.65:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:08,072] - [rest_client] [123145341620224] - INFO - settings/indexes params : storageMode=plasma
[2020-02-25 19:27:08,328] - [task] [123145341620224] - INFO - server: ip:172.23.121.67 port:8091 ssh_username:root, nodes/self: {'uptime': '26209', 'memoryTotal': 4366938112, 'memoryFree': 3800625152, 'mcdMemoryReserved': 3331, 'mcdMemoryAllocated': 3331, 'status': 'healthy', 'hostname': '172.23.121.67:8091', 'clusterCompatibility': 458752, 'clusterMembership': 'active', 'version': '7.0.0-1288-enterprise', 'os': 'x86_64-unknown-linux-gnu', 'ports': [], 'availableStorage': [], 'storage': [<membase.api.rest_client.NodeDataStorage object at 0x10e084f50>], 'memoryQuota': 1832, 'moxi': 11211, 'memcached': 11210, 'id': 'ns_1@172.23.121.67', 'ip': '172.23.121.67', 'rest_username': '', 'rest_password': '', 'port': '8091', 'services': ['kv'], 'storageTotalRam': 4164, 'curr_items': 0}
[2020-02-25 19:27:08,375] - [rest_client] [123145341620224] - ERROR - GET http://172.23.121.67:8091/pools/default body:  headers: {'Content-Type': 'application/x-www-form-urlencoded', 'Authorization': 'Basic QWRtaW5pc3RyYXRvcjpwYXNzd29yZA==', 'Accept': '*/*'} error: 404 reason: unknown b'"unknown pool"' auth: Administrator:password
[2020-02-25 19:27:08,376] - [rest_client] [123145341620224] - INFO - pools/default params : memoryQuota=2231
[2020-02-25 19:27:08,461] - [rest_client] [123145341620224] - INFO - --> in init_cluster...Administrator,password,8091
[2020-02-25 19:27:08,462] - [rest_client] [123145341620224] - INFO - settings/web params on 172.23.121.67:8091:port=8091&username=Administrator&password=password
[2020-02-25 19:27:08,564] - [rest_client] [123145341620224] - INFO - --> status:True
[2020-02-25 19:27:08,566] - [remote_util] [123145341620224] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:27:08,917] - [remote_util] [123145341620224] - INFO - Connected to 172.23.121.67
[2020-02-25 19:27:10,868] - [remote_util] [123145341620224] - INFO - running command.raw on 172.23.121.67: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:10,945] - [remote_util] [123145341620224] - INFO - command executed successfully
[2020-02-25 19:27:11,031] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.67:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:11,118] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.67:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:11,119] - [rest_client] [123145341620224] - INFO - settings/indexes params : storageMode=plasma
[2020-02-25 19:27:11,386] - [task] [123145341620224] - INFO - server: ip:172.23.121.66 port:8091 ssh_username:root, nodes/self: {'uptime': '26188', 'memoryTotal': 4366938112, 'memoryFree': 3823288320, 'mcdMemoryReserved': 3331, 'mcdMemoryAllocated': 3331, 'status': 'healthy', 'hostname': '172.23.121.66:8091', 'clusterCompatibility': 458752, 'clusterMembership': 'active', 'version': '7.0.0-1288-enterprise', 'os': 'x86_64-unknown-linux-gnu', 'ports': [], 'availableStorage': [], 'storage': [<membase.api.rest_client.NodeDataStorage object at 0x10d3da790>], 'memoryQuota': 2231, 'moxi': 11211, 'memcached': 11210, 'id': 'ns_1@172.23.121.66', 'ip': '172.23.121.66', 'rest_username': '', 'rest_password': '', 'port': '8091', 'services': ['kv'], 'storageTotalRam': 4164, 'curr_items': 0}
[2020-02-25 19:27:11,426] - [rest_client] [123145341620224] - INFO - pools/default params : memoryQuota=2231
[2020-02-25 19:27:11,494] - [rest_client] [123145341620224] - INFO - --> in init_cluster...Administrator,password,8091
[2020-02-25 19:27:11,495] - [rest_client] [123145341620224] - INFO - settings/web params on 172.23.121.66:8091:port=8091&username=Administrator&password=password
[2020-02-25 19:27:11,589] - [rest_client] [123145341620224] - INFO - --> status:True
[2020-02-25 19:27:11,591] - [remote_util] [123145341620224] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:27:11,925] - [remote_util] [123145341620224] - INFO - Connected to 172.23.121.66
[2020-02-25 19:27:13,838] - [remote_util] [123145341620224] - INFO - running command.raw on 172.23.121.66: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:13,919] - [remote_util] [123145341620224] - INFO - command executed successfully
[2020-02-25 19:27:13,982] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.66:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:14,040] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.66:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:14,041] - [rest_client] [123145341620224] - INFO - settings/indexes params : storageMode=plasma
[2020-02-25 19:27:14,290] - [task] [123145341620224] - INFO - server: ip:172.23.121.68 port:8091 ssh_username:root, nodes/self: {'uptime': '26172', 'memoryTotal': 4366938112, 'memoryFree': 3792203776, 'mcdMemoryReserved': 3331, 'mcdMemoryAllocated': 3331, 'status': 'healthy', 'hostname': '172.23.121.68:8091', 'clusterCompatibility': 458752, 'clusterMembership': 'active', 'version': '7.0.0-1288-enterprise', 'os': 'x86_64-unknown-linux-gnu', 'ports': [], 'availableStorage': [], 'storage': [<membase.api.rest_client.NodeDataStorage object at 0x10d3ef850>], 'memoryQuota': 2231, 'moxi': 11211, 'memcached': 11210, 'id': 'ns_1@172.23.121.68', 'ip': '172.23.121.68', 'rest_username': '', 'rest_password': '', 'port': '8091', 'services': ['kv'], 'storageTotalRam': 4164, 'curr_items': 0}
[2020-02-25 19:27:14,345] - [rest_client] [123145341620224] - INFO - pools/default params : memoryQuota=2231
[2020-02-25 19:27:14,395] - [rest_client] [123145341620224] - INFO - --> in init_cluster...Administrator,password,8091
[2020-02-25 19:27:14,396] - [rest_client] [123145341620224] - INFO - settings/web params on 172.23.121.68:8091:port=8091&username=Administrator&password=password
[2020-02-25 19:27:14,501] - [rest_client] [123145341620224] - INFO - --> status:True
[2020-02-25 19:27:14,503] - [remote_util] [123145341620224] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:27:14,866] - [remote_util] [123145341620224] - INFO - Connected to 172.23.121.68
[2020-02-25 19:27:16,813] - [remote_util] [123145341620224] - INFO - running command.raw on 172.23.121.68: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:16,882] - [remote_util] [123145341620224] - INFO - command executed successfully
[2020-02-25 19:27:16,930] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.68:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:16,997] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.121.68:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:16,998] - [rest_client] [123145341620224] - INFO - settings/indexes params : storageMode=plasma
[2020-02-25 19:27:17,239] - [task] [123145341620224] - INFO - server: ip:172.23.104.28 port:8091 ssh_username:root, nodes/self: {'uptime': '26148', 'memoryTotal': 3867246592, 'memoryFree': 3028254720, 'mcdMemoryReserved': 2950, 'mcdMemoryAllocated': 2950, 'status': 'healthy', 'hostname': '172.23.104.28:8091', 'clusterCompatibility': 458752, 'clusterMembership': 'active', 'version': '7.0.0-1422-enterprise', 'os': 'x86_64-unknown-linux-gnu', 'ports': [], 'availableStorage': [], 'storage': [<membase.api.rest_client.NodeDataStorage object at 0x10d3da710>], 'memoryQuota': 1976, 'moxi': 11211, 'memcached': 11210, 'id': 'ns_1@172.23.104.28', 'ip': '172.23.104.28', 'rest_username': '', 'rest_password': '', 'port': '8091', 'services': ['index', 'kv', 'n1ql'], 'storageTotalRam': 3688, 'curr_items': 0}
[2020-02-25 19:27:17,287] - [rest_client] [123145341620224] - INFO - pools/default params : memoryQuota=1976
[2020-02-25 19:27:17,345] - [rest_client] [123145341620224] - INFO - --> in init_cluster...Administrator,password,8091
[2020-02-25 19:27:17,346] - [rest_client] [123145341620224] - INFO - settings/web params on 172.23.104.28:8091:port=8091&username=Administrator&password=password
[2020-02-25 19:27:17,441] - [rest_client] [123145341620224] - INFO - --> status:True
[2020-02-25 19:27:17,443] - [remote_util] [123145341620224] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:27:17,739] - [remote_util] [123145341620224] - INFO - Connected to 172.23.104.28
[2020-02-25 19:27:19,489] - [remote_util] [123145341620224] - INFO - running command.raw on 172.23.104.28: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:19,578] - [remote_util] [123145341620224] - INFO - command executed successfully
[2020-02-25 19:27:19,631] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.104.28:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:19,688] - [rest_client] [123145341620224] - INFO - /diag/eval status on 172.23.104.28:8091: True content: [7,0] command: cluster_compat_mode:get_compat_version().
[2020-02-25 19:27:19,689] - [rest_client] [123145341620224] - INFO - settings/indexes params : storageMode=plasma
[2020-02-25 19:27:19,930] - [basetestcase] [123145331109888] - INFO - **** add built-in 'cbadminbucket' user to node 172.23.121.65 ****
[2020-02-25 19:27:20,103] - [rest_client] [123145331109888] - ERROR - DELETE http://172.23.121.65:8091/settings/rbac/users/local/cbadminbucket body:  headers: {'Content-Type': 'application/x-www-form-urlencoded', 'Authorization': 'Basic QWRtaW5pc3RyYXRvcjpwYXNzd29yZA==', 'Accept': '*/*'} error: 404 reason: unknown b'"User was not found."' auth: Administrator:password
[2020-02-25 19:27:20,112] - [internal_user] [123145331109888] - INFO - Exception while deleting user. Exception is -b'"User was not found."'
[2020-02-25 19:27:20,306] - [basetestcase] [123145331109888] - INFO - sleep for 5 secs.  ...
[2020-02-25 19:27:25,311] - [basetestcase] [123145331109888] - INFO - **** add 'admin' role to 'cbadminbucket' user ****
[2020-02-25 19:27:25,502] - [basetestcase] [123145331109888] - INFO - done initializing cluster
[2020-02-25 19:27:25,504] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:27:25,811] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:27:27,671] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: curl http://Administrator:password@localhost:8091/diag/eval -X POST -d 'ns_config:set(allow_nonlocal_eval, true).'
[2020-02-25 19:27:27,747] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:27:27,748] - [basetestcase] [123145331109888] - INFO - Enabled diag/eval for non-local hosts from 172.23.121.65
[2020-02-25 19:27:28,109] - [rest_client] [123145341620224] - INFO - http://172.23.121.65:8091/pools/default/buckets with param: name=default&authType=sasl&saslPassword=None&ramQuotaMB=2231&replicaNumber=1&bucketType=membase&replicaIndex=1&threadsNumber=3&flushEnabled=1&evictionPolicy=valueOnly&compressionMode=passive
[2020-02-25 19:27:28,194] - [rest_client] [123145341620224] - INFO - 0.08 seconds to create bucket default
[2020-02-25 19:27:28,194] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:28,717] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:29,304] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:29,642] - [task] [123145341620224] - WARNING - vbucket map not ready after try 0
[2020-02-25 19:27:29,642] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:30,111] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:30,689] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:31,082] - [task] [123145341620224] - WARNING - vbucket map not ready after try 1
[2020-02-25 19:27:31,083] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:31,611] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:32,236] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:32,545] - [task] [123145341620224] - WARNING - vbucket map not ready after try 2
[2020-02-25 19:27:32,546] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:33,045] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:33,618] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:33,954] - [task] [123145341620224] - WARNING - vbucket map not ready after try 3
[2020-02-25 19:27:33,955] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:34,425] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:34,933] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:35,250] - [task] [123145341620224] - WARNING - vbucket map not ready after try 4
[2020-02-25 19:27:35,251] - [bucket_helper] [123145341620224] - INFO - waiting for memcached bucket : default in 172.23.121.65 to accept set ops
[2020-02-25 19:27:35,740] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:36,313] - [data_helper] [123145341620224] - INFO - creating direct client 172.23.121.65:11210 default
[2020-02-25 19:27:36,664] - [task] [123145341620224] - WARNING - vbucket map not ready after try 5
[2020-02-25 19:27:36,677] - [basetestcase] [123145331109888] - INFO - ==============  basetestcase setup was finished for test #1 test_role_permission_validate_multiple ==============
[2020-02-25 19:27:36,883] - [basetestcase] [123145331109888] - INFO - ------- Cluster statistics -------
[2020-02-25 19:27:36,884] - [basetestcase] [123145331109888] - INFO - 172.23.121.65:8091 => {'services': ['kv'], 'cpu_utilization': 3.03030303030303, 'mem_free': 3132301312, 'mem_total': 4366938112, 'swap_mem_used': 43032576, 'swap_mem_total': 3758092288}
[2020-02-25 19:27:36,885] - [basetestcase] [123145331109888] - INFO - --- End of cluster statistics ---
[2020-02-25 19:27:37,133] - [rbacmain] [123145331109888] - INFO -  Deleting User - Status - True -- Content - b'""' -- Header - {'Cache-Control': 'no-cache,no-store,must-revalidate', 'Content-Length': '2', 'Content-Type': 'application/json', 'Date': 'Wed, 26 Feb 2020 03:27:36 GMT', 'Expires': 'Thu, 01 Jan 1970 00:00:00 GMT', 'Pragma': 'no-cache', 'Server': 'Couchbase Server', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'DENY', 'X-Permitted-Cross-Domain-Policies': 'none', 'X-XSS-Protection': '1; mode=block', 'status': '200'}
[2020-02-25 19:27:37,288] - [rbacmain] [123145331109888] - INFO -  Retrieve User Roles - Status - True -- Content - b'[]' -- Header - {'Cache-Control': 'no-cache,no-store,must-revalidate', 'Content-Type': 'application/json', 'Date': 'Wed, 26 Feb 2020 03:27:37 GMT', 'Expires': 'Thu, 01 Jan 1970 00:00:00 GMT', 'Pragma': 'no-cache', 'Server': 'Couchbase Server', 'Transfer-Encoding': 'chunked', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'DENY', 'X-Permitted-Cross-Domain-Policies': 'none', 'X-XSS-Protection': '1; mode=block', 'status': '200', 'content-location': 'http://172.23.121.65:8091//settings/rbac/users'}
[2020-02-25 19:27:37,307] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:27:37,676] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:27:39,717] - [rbacmain] [123145331109888] - INFO -  Value of e is [Errno 2] No such file
[2020-02-25 19:27:39,737] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:27:40,092] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:27:42,232] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:27:42,546] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:27:44,354] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: service saslauthd restart
[2020-02-25 19:27:44,506] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:27:44,517] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:27:44,867] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.67
[2020-02-25 19:27:46,871] - [rbacmain] [123145331109888] - INFO -  Value of e is [Errno 2] No such file
[2020-02-25 19:27:46,888] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:27:47,193] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.67
[2020-02-25 19:27:49,198] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.67 with username:root 
[2020-02-25 19:27:49,547] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.67
[2020-02-25 19:27:51,348] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.67: service saslauthd restart
[2020-02-25 19:27:51,557] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:27:51,567] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:27:51,915] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.66
[2020-02-25 19:27:54,041] - [rbacmain] [123145331109888] - INFO -  Value of e is [Errno 2] No such file
[2020-02-25 19:27:54,058] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:27:54,408] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.66
[2020-02-25 19:27:56,392] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.66 with username:root 
[2020-02-25 19:27:56,709] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.66
[2020-02-25 19:27:58,495] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.66: service saslauthd restart
[2020-02-25 19:27:58,610] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:27:58,620] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:27:58,968] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.68
[2020-02-25 19:28:00,944] - [rbacmain] [123145331109888] - INFO -  Value of e is [Errno 2] No such file
[2020-02-25 19:28:00,965] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:28:01,322] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.68
[2020-02-25 19:28:03,257] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.68 with username:root 
[2020-02-25 19:28:03,608] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.68
[2020-02-25 19:28:07,060] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.68: service saslauthd restart
[2020-02-25 19:28:07,407] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:28:07,418] - [remote_util] [123145331109888] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:28:08,166] - [remote_util] [123145331109888] - INFO - Connected to 172.23.104.28
[2020-02-25 19:28:10,061] - [rbacmain] [123145331109888] - INFO -  Value of e is [Errno 2] No such file
[2020-02-25 19:28:10,078] - [remote_util] [123145331109888] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:28:10,389] - [remote_util] [123145331109888] - INFO - Connected to 172.23.104.28
[2020-02-25 19:28:12,419] - [remote_util] [123145331109888] - INFO - connecting to 172.23.104.28 with username:root 
[2020-02-25 19:28:12,715] - [remote_util] [123145331109888] - INFO - Connected to 172.23.104.28
[2020-02-25 19:28:14,470] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.104.28: service saslauthd restart
[2020-02-25 19:28:14,615] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:28:14,674] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:28:15,019] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:28:16,818] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: ldapdelete -h 172.23.120.205 -p 389 cn=ritam,ou=Users,dc=couchbase,dc=com -D cn=Manager,dc=couchbase,dc=com -w p@ssword
[2020-02-25 19:28:16,925] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:28:16,926] - [remote_util] [123145331109888] - ERROR - bash: ldapdelete: command not found
[2020-02-25 19:28:16,928] - [remote_util] [123145331109888] - INFO - connecting to 172.23.121.65 with username:root 
[2020-02-25 19:28:17,246] - [remote_util] [123145331109888] - INFO - Connected to 172.23.121.65
[2020-02-25 19:28:19,082] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: echo "dn: cn=ritam,ou=Users,dc=couchbase,dc=com
cn:ritam
sn: ritam
objectClass: inetOrgPerson
userPassword :password
uid: ritam
" > /tmp/name.ldif
[2020-02-25 19:28:19,187] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:28:19,188] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: ldapadd -h 172.23.120.205 -p 389 -f /tmp/name.ldif -D cn=Manager,dc=couchbase,dc=com -w p@ssword
[2020-02-25 19:28:19,486] - [remote_util] [123145331109888] - INFO - command executed successfully
[2020-02-25 19:28:19,487] - [remote_util] [123145331109888] - ERROR - bash: ldapadd: command not found
[2020-02-25 19:28:19,488] - [remote_util] [123145331109888] - INFO - running command.raw on 172.23.121.65: rm -rf /tmp/*.ldif
[2020-02-25 19:28:19,658] - [remote_util] [123145331109888] - INFO - command executed successfully
